src/ci_workflow/ci_check_npm_package_version.py

```diff
@@ -13,9 +13,9 @@ class CiCheckNpmPackageVersion(CiCheckPackage):
     @property
     def checked_version(self) -> str:
         if self.component.name == "OpenSearch-Dashboards":
-            return self.target.opensearch_version.replace("-SNAPSHOT", "")
+            return self.target.opensearch_version.split("-")[0]
```

Comment:
If you split using `-` the opensearch version for `1.0.0-aplha1-snapshot` will be returned as `1.0.0` instead of `1.0.0-alpha1` is this expected? Also this is npm_package module, why is it returning `self.target.opensearch_version` instead of dashboards version

Reply:
This is expected as stated in my issue #1862.
Dashboards currently does not have value to compare with alpha1/snapshot.